### PR TITLE
Corresponds to migration of proc object

### PR DESCRIPTION
**Pull request by pyama86**
Hi!
ｍruby-thread is a great product!
We also want to deal with the migration of proc objects, so please consider them.
thanks.

### CODE REVIEW

File: test/thread.rb [hunk 1]
```diff
@@ -37,13 +37,16 @@
 end
 
 assert('Thread returns Array') do
-  a = Thread.new{[1,2,3]}
```
Comment (pyama86) [inline test/thread.rb]:
```ruby
a = Thread.new{[1, 2, 3]} 
puts a.join
=> Segmentation fault
```
refs: https://github.com/mattn/mruby-thread/issues/15

**Comment by mattn** [comment]
You should uncomment in mrbgems.rake to enable `MRB_THREAD_COPY_VALUES`.

**Comment by mattn** [comment]
merged https://github.com/mattn/mruby-thread/commit/72722af72b9af6925502630feb734715ae691532

**Comment by pyama86** [comment]
thanks!

**Pull request closed without merging**